doc/src/manual/constructors.md

```diff
@@ -114,9 +114,11 @@ violate this invariant. Of course, messing around with an object's internals uni
 You (or someone else) can also provide additional outer constructor methods at any later point, but
 once a type is declared, there is no way to add more inner constructor methods. Since outer constructor
 methods can only create objects by calling other constructor methods, ultimately, some inner constructor
-must be called to create an object. This guarantees that all objects of the declared type must come into
-existence by a call to one of the inner constructor methods provided with the type, thereby giving
-some degree of enforcement of a type's invariants.
+must be called to create an object. This means that usually at any given moment all existing objects of
+the declared type have come into existence by a call to one of the inner constructor methods provided
+with the type, thereby giving some degree of enforcement of a type's invariants. An exception to this
+rule are objects constructed using [Incomplete Initialization](@ref), which may bypass the inner
+constructor methods of a type.
```

Comment:
How about something like:
```suggestion
must be called to create an object. This means that usually at any given moment all existing objects of
the declared type have come into existence by a call to one of the inner constructor methods provided
with the type, thereby giving some degree of enforcement of a type's invariants. An exception to this
rule are "plain data" types created by bypassing the constructor, e.g. from uninitialized memory, such as a field in an [incompletely initialized](@ref incomplete-initialization) type, or via [`undef`](@ref) array constructor, or by [`reinterpret`](@ref).
```
(need to check the references work).

Reply:
What do you think about the latest commit? It's somewhat reworded and `undef` and `reinterpret` are xrefed now.